proto/storage/image_integration.proto

```diff
@@ -131,11 +135,13 @@ message ScannerV4Config {
   string matcher_endpoint = 3;
 }
 
+// Azure container registry configuration. Used by integrations of type "azure".
 message AzureConfig {
-  string endpoint = 1; // @gotags: scrub:"dependent" validate:"nolocalendpoint"
```

Comment:
Could we have an existing `endpoint` stored for Azure that contains some secret? I'm not sure how this `dependent` scrubbing option works.

Reply:
`dependent` does not scrub the field itself, it just makes sure that when the endpoint is changed, one also has to change the password. I put the tag back and called out in the UI that WIF overwrites existing credentials.